### PR TITLE
Project migrated under kubernetes-sigs, change import path prefix to sigs.k8s.io/descheduler

### PR DESCRIPTION
**Pull request by ingvagabund**
See also https://github.com/kubernetes/org/issues/1176

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@ingvagabund  thanks. Is it ready to be merged?

/approve

**Comment by ingvagabund** [comment]
@aveshagarwal debugging failing tests in travis

**Comment by aveshagarwal** [comment]
> @aveshagarwal debugging failing tests in travis

Thanks. Please let me know once it is ready and i will merge it.

**Comment by ingvagabund** [comment]
@aveshagarwal ready

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ before_script:
   - kind load docker-image kubernetes/pause
   - cp "$(kind get kubeconfig-path --name="kind")" /tmp/admin.conf
 script:
+- mkdir -p ~/gopath/src/sigs.k8s.io/
+- mv ~/gopath/src/github.com/kubernetes-sigs/descheduler ~/gopath/src/sigs.k8s.io/.
```
Comment (aveshagarwal) [inline .travis.yml]: is the dot at the end intentional? just asking does not matter much.

Comment (ingvagabund) [reply .travis.yml]: Just my convention. I can remove it if needed.

Comment (aveshagarwal) [reply .travis.yml]: i am fine, have not seen that frequently (for current dir at the end of the path)  so asking.

**Pull request merged**